docs/semistable_reduction.rst

```diff
@@ -2,7 +2,7 @@ Semistable reduction of curves
 ==============================
 
 .. toctree::
-    reduction_trees
-    superp
```

Comment:
@swewers Should the superp and superell files not be in the documentation at all anymore?